### ISSUE TITLE
Can't process more than 16 files

### ISSUE THREAD

**Issue by matthewmorek** [ferdinandtorggler/remove-svg-properties#2]
I just found out that `remove-svg-properties` can't process more than 16 files in any given directory; the execution simply stops after that, without any error, as if there was only ever 16 files in the source directory.

I've double checked permissions, and tried the tool using different permission settings on the source files/dir, to no effect.

Here's is a sample output:

``` bash
$ ls icons/ui/src | wc -l
      52
$ grunt 'remove-svg-properties'
Running "remove-svg-properties:all" (remove-svg-properties) task
account-circle.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
add-circle.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
announcement.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
archive.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
attach-file.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
attach-money.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
backspace.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
block.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
bug-report.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
cached.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
checkbox-checked.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
checkbox-unchecked.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
clear.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
credit-card.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
delete.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
done.colors-light-dark.svg: 0 inline styles, 0 attributes, 0 stylesheet rules removed.
```

My Gruntfile configuration:

``` javascript
'remove-svg-properties': {
  options: {
    properties: [rsp.PROPS_FILL],
    namespaces: ['i', 'sketch', 'inkscape']
  },
  all: {
    src: 'icons/ui/src/*.svg',
    dest: 'icons/ui/src'
  }
}
```

**Comment by everthis** [comment]
same here

**Comment by ferdinandtorggler** [comment]
Fixed it in v0.3.2
